RoundingDifferences.R

```diff
@@ -0,0 +1,66 @@
+library(dplyr)
```

Comment:
Please either remove this file or, if you think it would be useful for folks, let's make it into a 'normal' package function that lives in the `R/` folder. (Probably in a separate PR.)